environment.yml

```diff
@@ -30,6 +30,7 @@ dependencies:
  - numpy
  - pandas
  - progressbar
+ - pybind11
```

Comment:
@cmarshak I don't think pybind11 is needed for Rosetta2 emulation? I'm running Rosetta on M1 and it doesn't look to me like I have pybind11 installed. I also do editable pip install.